services/app-web/src/hooks/useTealium.ts

```diff
@@ -101,15 +101,13 @@ const useTealium = (): {
     useEffect(() => {
         // Do not add tealium for local dev or review apps
         if (process.env.REACT_APP_AUTH_MODE !== 'IDM') {
-            console.log(`mock tealium page view: ${tealiumPageName}`)
+            // console.log(`mock tealium page view: ${tealiumPageName}`)
```

Comment:
I commented out these since they are very useful when debugging tealium but add a lot of noise otherwise

I removed the progrmming error console.error because this will log in tests that are using useTealium and that seems excessive (but also not really worth mocking).